torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2236,7 +2235,20 @@ Tensor* tensorexpr::computeOperandValue(
             return a * clamp(zero, six, a + three) / six;
           });
     } break;
-
+    case aten::hardshrink: {
+      return computeTwoOperand(
+          "aten_hardshrink",
+          inputs,
+          outputShape,
+          outputType,
+          [](const ExprHandle& a, const ExprHandle& lambd) {
+            auto pos_clambd = Cast::make(a.dtype(), lambd);
+            auto neg_clambd = ExprHandle(-0) - pos_clambd;
```

Comment:
i'm not sure if `ExprHandle(-0)` is what we want, since I don't know what the dtype is on that. If the default dtype for that wider precision than `pos_clambd` output dtype may not be right. Can you use `Cast::make(0)` for the zero?

Reply:
this is how NNC implements negation, so it should probably keep the output in the same dtype. Let's double check with @bertmaher

Reply:
I mean can you do `Cast::make(a.dtype(), ExprHandle(-0)) - pos_clambd` ?